frontend/lib/classes/data/inspection_location.dart

```diff
@@ -29,6 +29,34 @@ class InspectionLocation extends Data with WithImgHashes, WithLangText {
   @JsonKey(name: 'Ort')
   final String? ort;
 
+//TODO: die meisten hier werden wahrscheinlich keine strings sein, aber naja
+  @JsonKey(name: "Eigentuemer")
+  String? eigentuemer;
+  @JsonKey(name: "Bauwerkhoehe")
+  String? bauwerkhoehe;
+  @JsonKey(name: "Baujahr")
+  String? baujahr;
+  @JsonKey(name: "Ansprechpartner")
+  String? ansprechpartner;
+  @JsonKey(name: "Steigwegtyp")
+  String? steigwegtyp;
+  @JsonKey(name: "Schluessel")
+  String? schluessel;
+  @JsonKey(name: "Abschaltungen")
+  String? abschaltungen;
+  @JsonKey(name: "Steckdosen")
+  String? steckdosen;
+  @JsonKey(name: "WC")
+  String? wC;
+  @JsonKey(name: "Lagerraeume")
+  String? lagerraeume;
+  @JsonKey(name: "Steigschutzschluessel")
+  String? steigschutzschluessel;
+  @JsonKey(name: "X")
```

Comment:
@MbgBn sind das hier auch map-koordinaten? also längen unt breitengrad? 

weil dann würde ich das im backend schon in latLng koordinaten umwandeln (momentan fragt das backend eh die mapquestapi (http://open.mapquestapi.com/geocoding/v1/address) welche koordinaten die addresse hat siehe [code](https://github.com/Mastbau-FN/inspector/blob/380ce25d8937ac22dc2787aefe379e7a117dc254/backend/extern/location.js#L3), das würde ich dann nur machen wenn in der datenbank nichts steht)

Reply:
ups hab die falschen zeilen markiert

Reply:
Ja. Das sind bereits LatLon Koordinaten, allerdings in Dezimalgrad-Darstellung (WGS84, EPSG:4326). Mapquest sollte die eigentlich direkt darstellen können, ohne Umwandlung. Die Adresse musst du nicht extra georeferenzieren, jeder Standort (also jede Inspektion) hat somit bereits die korrekten Koordinaten.